packages/lib-classifier/src/components/Classifier/Classifier.spec.js

```diff
@@ -214,7 +245,141 @@ describe('Components > Classifier', function () {
+      })
+      sinon.stub(panoptes, 'get').callsFake((endpoint, query, headers) => {
+        switch (endpoint) {
+          case '/field_guides': {
+            const field_guides = []
+            return Promise.resolve({ body: { field_guides }})
+          }
+          case '/project_preferences': {
+            const project_preferences = []
+            return Promise.resolve({ body: { project_preferences }})
+          }
+          case '/project_roles': {
+            const project_roles = [{ roles }]
+            return Promise.resolve({ body: { project_roles }})
+          }
+          case '/subjects/queued': {
+            const subjects = [subjectSnapshot, ...Factory.buildList('subject', 9)]
+            return Promise.resolve({ body: { subjects }})
+          }
+        }
+      })
+      sinon.stub(panoptes, 'post').callsFake((...args) => {
+        return Promise.resolve({ headers: {}, body: { project_preferences: []}})
+      })
+      const checkBearerToken = sinon.stub().callsFake(() => Promise.resolve('mockAuth'))
+      const checkCurrent = sinon.stub().callsFake(() => Promise.resolve({ id: 123, login: 'mockUser' }))
+      const authClient = { ...defaultAuthClient, checkBearerToken, checkCurrent }
+      const client = { ...defaultClient, panoptes }
+      const store = RootStore.create({}, { authClient, client })
+      const { rerender } = render(
+        <Classifier
+          classifierStore={store}
+          project={projectSnapshot}
+          workflowID={workflowSnapshot.id}
+          workflowSnapshot={workflowSnapshot}
+        />,
+        {
+          wrapper: withStore(store)
+        }
+      )
+      await when(() => store.subjectViewer.loadingState === asyncStates.success)
+      const newSnapshot = {
+        ...workflowSnapshot,
+        version: '0.1',
+        strings: {
+          ...workflowSnapshot.strings,
+          'tasks.T0.answers.0.label': 'Answer one',
+          'tasks.T0.answers.1.label': 'Answer two'
+        },
+        tasks: {
+          ...workflowSnapshot.tasks,
+          T0: {
+            ...workflowSnapshot.tasks.T0,
+            answers: [{ label: 'Answer one', next: 'T1' }, { label: 'Answer two', next: 'T2' }]
+          }
+        }
+      }
+      rerender(
+        <Classifier
+          classifierStore={store}
+          project={projectSnapshot}
+          workflowID={newSnapshot.id}
+          workflowSnapshot={newSnapshot}
+        />,
+        {
+          wrapper: withStore(store)
+        }
+      )
+      workflow = store.workflows.active
+      await when(() => store.subjectViewer.loadingState === asyncStates.success)
+      taskTab = screen.getByRole('tab', { name: 'TaskArea.task'})
+      tutorialTab = screen.getByRole('tab', { name: 'TaskArea.tutorial'})
+      subjectImage = screen.getByRole('img', { name: `Subject ${subjectSnapshot.id}` })
+      tabPanel = screen.getByRole('tabpanel', { name: '1 Tab Contents'})
+      const task = workflowSnapshot.tasks.T0
```

Comment:
I missed this. After the workflow version changes, the task panel should be showing tasks from `newSnapshot`, not `workflowSnapshot`. These tests are testing that the displayed workflow version hasn't changed. Fixed in #3239.